src/frontend/README.md

```diff
@@ -1,131 +1,34 @@
-# Frontend service
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-The **frontend** service is responsible for rendering the UI for the store's website.
-It serves as the main entry point for the application routing requests to their
-appropriate backend services.
-The application uses Server Side Rendering (SSR) to generate HTML consumed by
-the clients, which could be web browsers, web crawlers, mobile clients or something
-else.
+## Getting Started
 
-## OpenTelemetry instrumentation
+First, run the development server:
 
-### Initialization
-
-The OpenTelemetry SDK is initialized in `main` using the `InitTraceProvider` function.
-
-```go
-func InitTracerProvider() *sdktrace.TracerProvider {
-    ctx := context.Background()
-
-    exporter, err := otlptracegrpc.New(ctx)
-    if err != nil {
-        log.Fatal(err)
-    }
-    tp := sdktrace.NewTracerProvider(
-        sdktrace.WithSampler(sdktrace.AlwaysSample()),
-        sdktrace.WithBatcher(exporter),
-    )
-    otel.SetTracerProvider(tp)
-    otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{}))
-    return tp
-}
-```
-
-Services should call `TraceProvider.Shutdown()` when the service is shutdown to
-ensure all spans are exported.
-This service makes that call as part of a deferred function in `main`.
-
-```go
-    // Initialize OpenTelemetry Tracing
-    tp := InitTracerProvider()
-    defer func() {
-        if err := tp.Shutdown(context.Background()); err != nil {
-            log.Printf("Error shutting down tracer provider: %v", err)
-        }
-    }()
+```bash
+npm run dev
+# or
+yarn dev
 ```
 
-### HTTP instrumentation
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```

Comment:
This may be misleading, as if we deploy the whole demo, port `3000` is used by Grafana.

Reply:
I'm updating the readme to match what we have tomorrow morning